docs/source/operations/projections/healpix.rst

```diff
@@ -51,6 +51,12 @@ Parameters
 
 .. note:: All parameters for the projection are optional.
 
+.. option:: +rot
+
+    Rotation of the HEALPix map in degrees.
```

Comment:
I believe it should be explained that the rotation is done in the cartesian / projected coordinate space, and what is the convention for the orientation (trigonometric ? clockwise ?)